docs/architecture/ADRs/architectural_decision_records.md

```diff
@@ -0,0 +1,13 @@
+# Architectural Decision Records (ADRs)
```

Comment:
ADRs should have an ID number with them as well as a date created. This allows you to follow how decisions change over time. Normally the ID number would go at the start of the file name so that they're ordered correctly in the repo. Can you update these with an ID and date please.